Source/WebCore/loader/cache/CachedFontLoadRequest.h

```diff
@@ -94,6 +94,10 @@ class CachedFontLoadRequest final : public FontLoadRequest, public CachedFontCli
 
     void fontLoaded(CachedFont& font) final
     {
+        if (m_fontLoadedProcessed)
+            return;
+
+        m_fontLoadedProcessed = true;
```

Comment:
Should we wait to initialize this until `m_fontLoadRequestClient->fontLoaded(*this)` returns?